database/migrations/2017_07_17_140109_add_payment_message_fieldToMarket.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
A bit late but you can generate with command next time. https://laravel.com/docs/5.4/migrations#generating-migrations

 `php artisan make:migration add_votes_to_users_table --table=users`